tests/pr_test.go

```diff
@@ -137,7 +137,7 @@ func TestFSCloudInSchematic(t *testing.T) {
 
 	// If "jp-osa" was the best region selected, default to us-south instead.
 	// "jp-osa" is currently not allowing hs-crypto be used for encrypting in that region.
```

Comment:
You you say why we skip us-east. Point to this provider issue: https://github.com/IBM-Cloud/terraform-provider-ibm/issues/4843